### PR TITLE
BridgeConstants For Period Between UTXOs Recycling (Timestamp And Blo…

### PR DESCRIPTION
**Pull request by kelvinator07**

### CODE REVIEW

**Review by josedahlquist — APPROVED** [review]

File: rskj-core/src/main/java/co/rsk/config/BridgeConstants.java [hunk 1]
```diff
@@ -78,6 +78,10 @@
 
     protected int minimumPegoutValuePercentageToReceiveAfterFee;
 
+    protected Integer utxoExpirationCheckpointRskBlockHeight;
+    protected Long utxoExpirationCheckpointTimestamp;
```
Comment (marcos-iov) [inline rskj-core/src/main/java/co/rsk/config/BridgeConstants.java:82]:
```suggestion
    protected Long utxoExpirationCheckpointPeriodTimestamp;
```

File: rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java [hunk 2]
```diff
@@ -157,5 +157,9 @@ public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
         maxDepthBlockchainAccepted = 25;
 
         minimumPegoutValuePercentageToReceiveAfterFee = 20;
+
+        utxoExpirationCheckpointRskBlockHeight = 40000; // rounded 2 weeks of RSK blocks
```
Comment (marcos-iov) [inline rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java:161]:
```suggestion
        utxoExpirationCheckpointRskBlockHeight = 40_000; // rounded 2 weeks of RSK blocks
```
Makes it clearer, some in other files

File: rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java [hunk 3]
```diff
@@ -157,5 +157,9 @@ public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
...
+        utxoExpirationCheckpointTimestamp = 1209600000L; // 2 weeks timestamp in milliseconds.
```
Comment (marcos-iov) [inline rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java:162]:
```suggestion
        utxoExpirationCheckpointTimestamp = 1_209_600_000L; // 2 weeks timestamp in milliseconds.
```

File: rskj-core/src/main/java/co/rsk/config/BridgeConstants.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (kelvinator07) [reply rskj-core/src/main/java/co/rsk/config/BridgeConstants.java]: It Should Be int and long.. Resolved

File: rskj-core/src/main/java/co/rsk/config/BridgeConstants.java [hunk 4]
```diff
@@ -78,6 +78,10 @@
...
+    protected int maxUtxoToRecycle;
```
[the remark this answers is not in the document]
Comment (kelvinator07) [reply rskj-core/src/main/java/co/rsk/config/BridgeConstants.java]: Done

File: rskj-core/src/main/java/co/rsk/config/BridgeConstants.java [hunk 5]
```diff
@@ -78,6 +78,10 @@
 
     protected int minimumPegoutValuePercentageToReceiveAfterFee;
 
+    protected Integer utxoExpirationCheckpointRskBlockHeight;
```
[the remark this answers is not in the document]
Comment (kelvinator07) [reply rskj-core/src/main/java/co/rsk/config/BridgeConstants.java]: Done

File: rskj-core/src/main/java/co/rsk/config/BridgeConstants.java (hunk 1 above)
Comment (kelvinator07) [reply rskj-core/src/main/java/co/rsk/config/BridgeConstants.java]: Done

File: rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java (hunk 2 above)
Comment (kelvinator07) [reply rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java]: Resolved

File: rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java (hunk 3 above)
Comment (kelvinator07) [reply rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java]: Resolved

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**